### PR TITLE
Add CI for release/uwp6.0 branches of coreclr and corefx

### PR DESCRIPTION
**Pull request by zamont**

### CODE REVIEW

**Comment by zamont** [comment]
@mmitche PTAL

**Comment by joshfree** [comment]
LGTM

**Comment by zamont** [comment]
@dotnet/dnceng could someone take a look and merge if it looks good?

File: jobs/data/repolist.txt [hunk 1]
```diff
@@ -48,11 +49,13 @@ jorive/coreclr branch=dev/baseline-with-new-api server=dotnet-ci2 definitionScri
 dotnet/coreclr branch=release/1.0.0 server=dotnet-ci
 dotnet/coreclr branch=release/1.1.0 server=dotnet-ci
 dotnet/coreclr branch=release/2.0.0 server=dotnet-ci
+dotnet/coreclr branch=release/uwp6.0 server=dotnet-ci
 dotnet/coreclr branch=dev/unix_test_workflow server=dotnet-ci
 dotnet/corefx branch=master server=dotnet-ci
 dotnet/corefx branch=release/1.0.0 server=dotnet-ci
 dotnet/corefx branch=release/1.1.0 server=dotnet-ci
 dotnet/corefx branch=release/2.0.0 server=dotnet-ci
+dotnet/corefx branch=release/uwp6.0 server=dotnet-ci
```
Comment (maririos) [inline jobs/data/repolist.txt]: I don't have context on how we want the UWP work to run, so just make sure that we do want them to use `dotnet-ci` and not the pipeline jobs like corefx release/2.0 does (`dotnet-ci3`)

Comment (maririos) [inline jobs/data/repolist.txt]: @joperezr could you help us here please?

**Comment by maririos** [comment]
Otherwise, LGTM

File: jobs/data/repolist.txt (hunk 1 above)
Comment (zamont) [inline jobs/data/repolist.txt]: @danmosemsft can you address Mariana's concern?

Comment (danmosemsft) [inline jobs/data/repolist.txt]: I think we would want them to do whatever master does, but I don't know. @weshaggard ?

Comment (weshaggard) [inline jobs/data/repolist.txt]: Yes do what master does. You will need to use dotnet-ci3 in order to use the helix test infra

**Comment by joshfree** [comment]
@karajas @dleeapho could someone from central engineering merge this when you signoff?

In addition to CI for the release/uwp6.0 branches, I believe we also need to enable Helix runs still.

Thanks!

**Comment by MattGal** [comment]
@joshfree Helix runs: https://mc.dot.net/#/product/netcore/uwp60

File: jobs/data/repolist.txt (hunk 1 above)
Comment (zamont) [inline jobs/data/repolist.txt]: I'm confused. Do any changes need to be made to this line? The line I added looks identical to the master branch line except with the branch being set to release/uwp6.0.

Comment (weshaggard) [inline jobs/data/repolist.txt]: This line can stay but you also need to add the following line:

`dotnet/corefx branch=release/uwp6.0 server=dotnet-ci3 definitionScript=pipelinejobs.groovy utilitiesRepoBranch=dev/pipelinesupport`

Comment (zamont) [inline jobs/data/repolist.txt]: Added.

Comment (zamont) [inline jobs/data/repolist.txt]: Do these changes look good now?

Comment (maririos) [inline jobs/data/repolist.txt]: Yes. Thanks Zach! I'll merge it then

**Comment by karajas** [comment]
Just following up, do we need anything else to turn this on?

**Comment by ellismg** [comment]
>  Just following up, do we need anything else to turn this on?

I don't believe so.  This should be sufficient.

**Comment by joshfree** [comment]
If you look at PRs like https://github.com/dotnet/corefx/pull/22557, you'll see that an insufficient amount of legs are running.  What else needs to happen to correct this?

**Comment by maririos** [comment]
@zamont  It looks like the changes are not in effect. After reading the pipelines [documentation](https://github.com/dotnet/dotnet-ci/blob/dev/pipelinesupport/docs/CI-PIPELINES.md) it looks like the change needs to happen in the branch `dev/pipelinesupport` and not `master`.

**Comment by zamont** [comment]
@maririos @weshaggard I created https://github.com/dotnet/dotnet-ci/pull/735 to port these changes to dev/pipelinesupport

**Comment by maririos** [comment]
FYI @smile21prc

**Comment by smile21prc** [comment]
#735 fails in check "Gen CI(dotnet-ci3) - dev/pipelinesupport/tests/dsl/*.groovy ". I'll take a look to find out why that fails.

**Comment by joshfree** [comment]
this hasn't resolved the CI leg issue still...

see https://github.com/dotnet/corefx/pull/22557

**Comment by smile21prc** [comment]
CI failures in #735 was fixed, but not all expected CI legs are run. I'll take a look on how to trigger ALL the expected CI legs.

**Comment by karajas** [comment]
Reopening the PR worked for future

**Comment by smile21prc** [comment]
CI leg issue seems to be resolved, and 11 legs now show in https://github.com/dotnet/corefx/pull/22557.

Four legs still fail. From the logs all failures seem to be test failures instead of Jenkins issues, and need to be investigated whether they are related to the PR changes themselves.

@zamont @joshfree , please go ahead take a first look at these failed legs to see whether they are caused by the PR changes. Feel free to let me know if you need any help from me.

**Comment by smile21prc** [comment]
Added first responder issue https://github.com/dotnet/core-eng/issues/1311 to track the work to resolve failed/missing new CI legs.

**Pull request merged**